### DISCUSSION TITLE
Support for Linux

### DISCUSSION THREAD

**Discussion by fadyanwar** [microsoft/Power-Fx#50] [Q&A] [answered]
Hi, I see the target framework in the csproj files is for netstandard2.0. Are there plans to use .net core so it would be possible to build and run on Linux?

**Comment by lesaltzm** [comment, marked as answer]
There shouldn't be anything in these projects that prevents running on linux, .NET Standard should allow you to consume this from a project running on Linux. Are you running into specific issues?

**Comment by fadyanwar** [comment]
Thanks for your reply. I've tried already building it on Ubuntu using .net core compiler and I got build errors even after modifying to csproj file to target .net core different frameworks instead. The samples using the Nugets however does build and run nicely. Are those Nugets built differently? i.e. different repo targeting .net core instead

**Reply by MikeStall** [reply]
Looks like you go this working ;) 
https://fadyanwar.com/index.php/2021/11/19/powerfx-linux-shell-scripting/

**Reply by fadyanwar** [reply]
Yes, I guess this question is answered now :)
